<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QWidget" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка игры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Симулятор инвестиций в строительство</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="playersLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество игроков (2-5):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="playerCountSpin">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="monthsLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Количество месяцев (6-24):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="monthsSpin">
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="nameLayout">
     <item>
      <layout class="QHBoxLayout" name="player1Layout">
       <item>
        <widget class="QLabel" name="player1Label">
         <property name="text">
          <string>Игрок 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="player1Edit">
         <property name="placeholderText">
          <string>Введите имя игрока 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="player2Layout">
       <item>
        <widget class="QLabel" name="player2Label">
         <property name="text">
          <string>Игрок 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="player2Edit">
         <property name="placeholderText">
          <string>Введите имя игрока 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="player3Layout">
       <item>
        <widget class="QLabel" name="player3Label">
         <property name="text">
          <string>Игрок 3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="player3Edit">
         <property name="placeholderText">
          <string>Введите имя игрока 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="player4Layout">
       <item>
        <widget class="QLabel" name="player4Label">
         <property name="text">
          <string>Игрок 4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="player4Edit">
         <property name="placeholderText">
          <string>Введите имя игрока 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="player5Layout">
       <item>
        <widget class="QLabel" name="player5Label">
         <property name="text">
          <string>Игрок 5:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="player5Edit">
         <property name="placeholderText">
          <string>Введите имя игрока 5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Начать игру</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>